lib/property_sets/active_record_extension.rb

```diff
@@ -154,20 +201,11 @@ def lookup_without_default(arg)
         detect { |property| property.name.to_sym == arg.to_sym }
       end
 
-      def lookup_value(type, key)
-        serialized = property_serialized?(key)
-
-        if instance = lookup_without_default(key)
-          instance.value_serialized = serialized
-          PropertySets::Casting.read(type, instance.value)
-        else
-          value = association_class.default(key)
-          if serialized
-            PropertySets::Casting.deserialize(value)
-          else
-            PropertySets::Casting.read(type, value)
-          end
-        end
```

Comment:
This logic has been moved into `PropertySets::ValueReader`, but I haven't changed it.